pyramus/updates/112.001/220926_staff-member-study-programme.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<update xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.ofw.fi/xml/2011/java-xmldb-updater/UpdaterSchema.xsd">
+
+  <sql>drop table if exists __StaffMemberStudyProgrammes;</sql>
```

Comment:
Not sure the drop is necessary here, I would probably prefer without.

Reply:
Not necessary as such - just there to avoid that awkward situation when a dev has already made said adjustments to their own dev database but later runs xmldbupdater to ensure that the database truly is up-to-date with all pending changes. Can be removed but can't see no real harm it being there.